complete/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+#spring.cloud.gcp.projectId=[YOUR_GCP_PROJECT_ID_HERE]
+#spring.cloud.gcp.credentialsLocation=file:[LOCAL_FS_CREDENTIALS_PATH]
+
+#spring.cloud.gcp.pubsub.subscriber.executorThreads=[NUM_SUBSCRIBER_THREADS]
```

Comment:
If you don't need threads to "introduce" people to this, I'd avoid it. Guides are supposed to be lunchtime readings.